provisioning/miniprov/hare_mp/main.py

```diff
@@ -647,6 +647,18 @@ def generate_support_bundle(args):
         if args.t:
             cmd.append('-t')
             cmd.append(args.t[0])
+        if args.duration:
+            noop(args.duration)
```

Comment:
`noop()` will force the application to exit immediately.
Which means that whenever the user specifies `--duration`, `--size-limit`, `--stacktrace` or whatever from the new requirements, the following things will happen:
1. No logs will be generated at all.
2. `hare_setup` will exit with 0 exit code.

Are you sure this is what we are expected to do?
"Ignoring some of the parameters" (why do we ignore them, BTW?) is not equal to "Do not collect the logs at all".

Reply:
Added logs in this function.

Reply:
okay, then why not have a version of `noop()` may be `not_implemented()` instead of faking the behavior. Anyway, I think its okay for now, not a big  thing to worry about for now.